### PR TITLE
Using build to build the sdist and wheel files

### PR DESCRIPTION
**Pull request by Usman2ABN**

### CODE REVIEW

**Comment by mbyrnepr2** [comment]
The latest CI failures seem to stem from the setup.cfg of the backend - Celery and its transitory dependencies install fine but every top-level dep after celery here is not installed. Looks like the [indentation is off](https://github.com/abnamro/repository-scanner/pull/200/files#diff-353086af6667cfc7c1c15149e7d7a606bac792c7e76ce8683df7473caaa25b25R16-R35).

Other than that - I think we shouldn't pin these dependencies so tightly like this, because then we run into what is discussed in https://github.com/abnamro/repository-scanner/pull/181. But we can open a separate issue to tackle that if you prefer (I envisage it to end up like the setup.cfg/setup.py of the wizard where a range of versions are acceptable and not one specific version for each dependency - which makes it more difficult to install other packages in the same environment as resc (which have their own dependency requirements).

File: components/resc-backend/setup.cfg [hunk 1]
```diff
@@ -11,7 +11,28 @@ long_description_content_type = text/markdown
...
+cryptography==41.0.3
+tenacity==8.2.2
+tomlkit==0.12.1
+python-multipart==0.0.6
+aiofiles==0.8.0
+GitPython==3.1.32
+packaging==23.1
+fastapi-cache2==0.2.1
+redis==4.6.0
+include_package_data = True
```
Comment (mbyrnepr2) [inline components/resc-backend/setup.cfg:35]:
```suggestion
    amqp==5.1.1
    requests==2.31.0
    sqlalchemy==2.0.20
    alembic==1.11.2
    fastapi==0.103.0
    typing==3.7.4.3
    pydantic==1.8.2
    uvicorn==0.17.6
    waitress==2.1.2
    pyjwt[crypto]==2.8.0
    cryptography==41.0.3
    tenacity==8.2.2
    tomlkit==0.12.1
    python-multipart==0.0.6
    aiofiles==0.8.0
    GitPython==3.1.32
    packaging==23.1
    fastapi-cache2==0.2.1
    redis==4.6.0
    include_package_data = True
```

File: components/resc-backend/setup.cfg [hunk 2]
```diff
@@ -11,7 +11,28 @@ long_description_content_type = text/markdown
...
+pyjwt[crypto]==2.8.0
+cryptography==41.0.3
+tenacity==8.2.2
+tomlkit==0.12.1
+python-multipart==0.0.6
+aiofiles==0.8.0
+GitPython==3.1.32
+packaging==23.1
+fastapi-cache2==0.2.1
+redis==4.6.0
```
Comment (mbyrnepr2) [inline components/resc-backend/setup.cfg:34]:
```suggestion
    amqp==5.1.1
    requests==2.31.0
    sqlalchemy==2.0.20
    alembic==1.11.2
    fastapi==0.103.0
    typing==3.7.4.3
    pydantic==1.8.2
    uvicorn==0.17.6
    waitress==2.1.2
    pyjwt[crypto]==2.8.0
    cryptography==41.0.3
    tenacity==8.2.2
    tomlkit==0.12.1
    python-multipart==0.0.6
    aiofiles==0.8.0
    GitPython==3.1.32
    packaging==23.1
    fastapi-cache2==0.2.1
    redis==4.6.0
```

File: components/resc-backend/setup.cfg (hunk 1 above)
Comment (mbyrnepr2) [inline components/resc-backend/setup.cfg:35]:
```suggestion
include_package_data = True
```
Is this change necessary? (similar for the other occurrences)

**Comment by mbyrnepr2** [comment]
FYI added https://github.com/abnamro/repository-scanner/issues/201 so we can separate the version-pinning refactor into a different PR.

> This should also fix/close #181 btw.

**Review by mbyrnepr2 — APPROVED** [review]

**Review by Peter-v-d-Spek — DISMISSED** [review]

**Review by Peter-v-d-Spek — APPROVED** [review]
